src/universal/modules/userDashboard/components/UserActionList/UserActionList.js

```diff
@@ -122,6 +131,8 @@ const styleThunk = () => ({
     borderWidth: '0 1px 1px',
     boxShadow: `inset 0 1px 0 ${ui.cardBorderColor}`,
     maxHeight: '100%',
+    // @terry this is required for no scroll bars, but conflicts with overflowTouch
```

Comment:
@mattkrick so I took a look at this overflow property. Was curious why it’s needed? I commented it out and didn’t see any adverse effects.

Currently the entire list is designed to be scrollable once it reaches at max-width at the bottom of the viewport. And adding that prop:value disables that functionality.

Just let me know—not urgent!